tools/scripts/validate_alerts.py

```diff
@@ -0,0 +1,39 @@
+import argparse
+import json
+import jsonschema
+
+ALERT_SCHEMA = {
+    "type": "array",
+    "items": {
```

Comment:
should the value of "items" be a list instead of dictionary?

Reply:
We're just defining a schema, so we're just defining the type of the objects in the array.
https://json-schema.org/understanding-json-schema/reference/array.html